ACMILabs/transcoder
# #1903 Allow disabling web copy creation

Partially resolves ACMILabs/xos#1903

This PR complements https://github.com/ACMILabs/xos/pull/2013 in solving ACMILabs/xos#1903.

- We can now tell the transcoder not to create web-optimised copies by setting `TRANSCODE_WEB_COPY=False` in the `dev.env` file

### Acceptance Criteria
- [x] Transcoder no longer creates web copies

### Relevant design files
* None

### Testing instructions
1. If it's your first time setting up the transcoder, ask David or Greg for a copy of the `dev.env` file, and other set up instructions.
2. Add a short video to the test watch folder `//fsqcollnas.corp.acmi.net.au/Working/xos-test/watch` with appropriate name e.g. `B0000123_mp01_testvideo.mp4`
3. Spin up a local instance of XOS and the transcoder in development mode: `docker-compose -f docker-compose-dev.yml up --build`
4. Check that the asset was created in your local XOS and that it does not have `web_resource` and `web_metadata`

### DoD
For requester to complete:
~New logic has been documented~
~New logic has appropriate unit tests~
~Changelog has been updated if necessary~
~Deployment / migration instruction have been updated if required~
